src/ui/game/GameScreen.js

```diff
@@ -48,17 +48,25 @@ type State = {
 };
 
 export default class GameScreen extends Component<Props, State> {
+  static getDerivedStateFromProps(props: Props, state: State) {
+    let { chatSections } = state;
+    let { game } = props;
+    let currentChatSections = getGameChatSections(game);
+
+    if (chatSections.length !== currentChatSections.length) {
+      return { ...state, chatSections: currentChatSections };
+    }
+
+    return null;
```

Comment:
return [] in this case to match previous behavior?

Reply:
Returning `null` in this case indicates that the component state shouldn't be changed from whatever it currently may be. It's a requirement of the `getDrivedStateFromProps` method: https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops

In other words, it's saying that no new chat messages have come in from the props, so the component state doesn't need to reflect that.

The `chatSections` state is being defaulted to an empty array on line 65, so it should always at least be that.

Reply:
I see, thanks!